Gruntfile.js

```diff
@@ -455,7 +503,7 @@ module.exports = (grunt) => {
     const esbuild = this.flags.watch ? 'esbuild:watch' : 'esbuild'
 
     if (!grunt.option('skipbuild')) {
-      grunt.task.run(['exec:eslint', 'exec:flow', 'exec:puglint', 'exec:stylelint', 'clean', 'copy', esbuild])
+      grunt.task.run(['exec:eslint', 'exec:flow', 'exec:puglint', 'exec:stylelint', 'clean:dist', 'copy:frontend', esbuild])
```

Comment:
As mentioned above, I think these changes can be reverted for both `'clean'` and `'copy'`